controllers/app.controller.js

```diff
@@ -43,3 +44,15 @@ exports.getArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+
+  readCommentsByArticleId(article_id)
+    .then((comment) => {
+      res.status(200).send({ comment });
```

Comment:
When returning an array of objects on a key, you should use the plural sense of the word, a user would expect to see comments rather than a singular comment, even if their is only 1 comment in the array.